README.md

```diff
@@ -117,12 +135,11 @@ IntelliJ IDEA -> Preferences -> Editor -> Inspections -> Settings icon -> Import
 
 ### Running cloudbreak in IDEA
 
-
 To launch the Cloudbreak application execute the `com.sequenceiq.cloudbreak.CloudbreakApplication` class  (Set 'Use classpath of module' to `core_main`) with the following VM options:
 ```
 -Dcb.client.id=cloudbreak
 -Dcb.client.secret=CB_SECRET_GENERATED_BY_CBD
--Dcb.db.port.5432.tcp.addr=localhost
+-Dcb.db.port.5432.tcp.addr=<DB_ADDRESS>
 -Dcb.db.port.5432.tcp.port=5432
 -Dcb.identity.server.url=http://localhost:8089
```

Comment:
This should be also the `localhost` os `bridge ip`

Reply:
Forgive me, which line? Line 142 is DB_ADDRESS and is explained by a blurb below it to be localhost or bridge ip.

Reply:
`-Dcb.identity.server.url=http://localhost:8089`

Reply:
This actually is not necessary -- only the database needs to use the docker bind address, nothing else.